clang/include/clang/Basic/Sanitizers.h

```diff
@@ -154,6 +154,8 @@ struct SanitizerKind {
 #include "clang/Basic/Sanitizers.def"
 }; // SanitizerKind
 
+using SanitizerMaskCutoffs = std::array<float, SanitizerKind::SO_Count>;
```

Comment:
I was thinking what could be  a better implementation to store values. I can be different depending how use those constants.

Let's create a special class for there. 
Maybe SanitizerMaskCutoffs?
and put array and NoSanitizeTopHot into that.

Reply:
Like this? https://github.com/llvm/llvm-project/pull/121619/commits/56ba349e7f085fca3bfd531ea219b1d9e8ef177c